herbert_core/utilities/misc/deserialise.m

```diff
@@ -3,33 +3,33 @@
 % the array of bytes using serialize routine.
 % Inputs:
 % a   -- array of bytes, contaning serialized objects
-% pos -- starting position of the data to deserialize. If missing, assumed that data to deserialize 
+% pos -- starting position of the data to deserialize. If missing, assumed that data to deserialize
 %        are located from the beginning of the input array
 % Outputs:
 % ser    -- deserialized contencts of the array of bytes
 % nbytes -- the extend, deserialized data were occupied in the input array
 %
 %
-if nargin<2
-    pos = 1;
-end
-[use_mex,fm] = config_store.instance().get_value('herbert_config',...
-    'use_mex','force_mex_if_use_mex');
-% Temporary disabled mex, #394
-use_mex = false;
+    if nargin<2
+        pos = 1;
+    end
+    [use_mex,fm] = config_store.instance().get_value('herbert_config',...
+                                                     'use_mex','force_mex_if_use_mex');
+    % Temporary disabled mex, #394
+    use_mex = false;
 
-if use_mex
-    try
-        [ser,nbytes] = c_deserialise(a,pos);
-        return
-    catch ME
-        if fm
-            rethrow(ME);
-        else
-            warning(ME.identifier,'%s',ME.message);
+    if use_mex
+        try
+            [ser,nbytes] = c_deserialise(a,pos);
+        catch ME
+            if fm
+                rethrow(ME);
+            else
+                warning(ME.identifier,'%s',ME.message);
+            end
```

Comment:
Do we still want to have use_mex as an option? I think it has been disabled for a long time. Just a thought, and worth checking with Alex.